apps/remote_control/lib/lexical/remote_control/build/parse_error.ex

```diff
@@ -0,0 +1,176 @@
+defmodule Lexical.RemoteControl.Build.ParseError do
+  alias Lexical.Document
+  alias Lexical.Plugin.V1.Diagnostic.Result
+
+  import Lexical.RemoteControl.Build.ErrorSupport,
```

Comment:
Not a fan of importing normal functions

Reply:
I'm curious How do you typically differentiate between normal functions and abnormal functions?

Reply:
as opposed to macros or sigils

Reply:
I don't import functions unless i'm in a test case.